docs/Основные блоки и устройство компьютера/Устройства вывода информации.md

```diff
@@ -1,39 +1,73 @@
-# Устройства вывода информации. 
-**Устройства вывода** — технические устройства, позволяющие получить данные от компьютера в виде, удобном для восприятия пользователя.
-Устройства вывода различают, в зависимости от типа выводимой из компьютера информации. Существует графическая, звуковая и прочие виды информации.
+## Устройства вывода информации
```

Comment:
[ch-md-error-first-line-h1](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-first-line-h1)